Automattic/wp-calypso
# Ensure Credits tab has its right margin

Related to SELFSERVE-553
Discussion: p1686559561135739-slack-C053FAERCDP

We want to make Tabs Navigation within Tools > Advertising accessible from mobile devices.

## Proposed Changes

* Hide dropdown menu (it isn't used but might be shown),
* Make sure the Credits tab is aligned to the right for mobile screen too,
* For small mobile screens (less than 364px; the number was chosen according to the current tab's content) make sure we can slide to tabs that no fully visible due to small screen size:

## Testing Instructions



* Check out to this branch and build assets
* Open Tools > Advertising
* Try various screen widths from `1300px` down to `364px` and make sure tabs are fully visible
* Check screen width of `364px` (this screen width should have all tabs visible
<img width="639" alt="Screenshot 2023-06-22 at 15 53 47" src="https://github.com/Automattic/wp-calypso/assets/115007291/e2dd170e-b4ab-4a91-bf36-ddb8d00330b2">

* Make screen smaller (e.g. `320px`)
* Ensure clicking to "Credits" tabs leaves `16px` margin to the right from it:

https://github.com/Automattic/wp-calypso/assets/115007291/f32a0922-ede7-42ee-ae4a-39d0e25afa84

## Pre-merge Checklist
